cinema/tests/test_movie_api.py

```diff
@@ -157,3 +157,165 @@ def test_image_url_is_shown_on_movie_session_detail(self):
         res = self.client.get(MOVIE_SESSION_URL)
 
         self.assertIn("movie_image", res.data[0].keys())
```

Comment:
The key 'movie_image' is being checked in the movie session detail response. Ensure that the API actually returns this key, as it might be different from the 'image' key used elsewhere.